### ISSUE TITLE
Create dbt converter for Airbyte Shopify source connector

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte-dbt-models#28]
The goal of this task is to ensure the Airbyte Shopify connector output can be used to run [Fivetran's dbt analytical model](https://hub.getdbt.com/fivetran/shopify/latest/).

**Definition of Done:** 
* Create a pull request that adds new tables or views, enabling the execution of Fiventran's dbt with Airbyte output data.
* Update the documentation inside the connector folder to explain how to configure the user dbt package to run the analytical model.

**Resources:** 
* Check example of how it was done for [Zendesk Support](link) connector

**Comment by avirajsingh7** [comment]
Hey @marcosmarxm , Can you assign me this one?

**Comment by marcosmarxm** [comment]
@avirajsingh7 any updates if not I'm gonna unassigned you.

**Comment by bala-ceg** [comment]
@marcosmarxm can u please assign this to me if the issue is available

**Comment by marcosmarxm** [comment]
@avirajsingh7 do you have an update? I know this connector there are a lot of to be done but need to update about the status.

**Comment by avirajsingh7** [comment]
@marcosmarxm not done much of work, you can assign this to @bala-ceg

**Comment by marcosmarxm** [comment]
@bala-ceg all yours.

**Comment by bala-ceg** [comment]
@marcosmarxm, Please review the step 1 

| Fivetran                     | Airbyte            |
| ---------------------------- | ------------------ |
| shopify__customer_cohorts     |   ❌ it doesn't exist                 |
| shopify__customers            |   Customers                 |
| shopify__customer_email_cohorts|      ❌ it doesn't exist             |
| shopify__customer_emails      |      ❌ it doesn't exist                |
| shopify__orders               |       Orders             |
| shopify__order_lines          |     ❌ it doesn't exist                  |
| shopify__products             |    Products                |
| shopify__transactions         |       Transcations             |
| shopify__daily_shop           |     ❌ it doesn't exist                  |
| shopify__discounts            |  Discount Codes               |
| shopify__inventory_levels     | Inventory Levels              |
| shopify__line_item_enhanced   |     ❌ it doesn't exist                  |

**Comment by marcosmarxm** [comment]
@bala-ceg can you check if the customer table has an object or array called cohors/emails?

**Comment by bala-ceg** [comment]
yes,@marcosmarxm. customer table has emails

**Comment by marcosmarxm** [comment]
No can you confirm all missing tables have objects in the parent table?

**Comment by bala-ceg** [comment]
Yes i have checked, those missing tables are not in any of the airbyte streams.
